TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,113 @@
+When used in first-party contexts, their semantics are identical to that of first-party cookies.
+
+### 2.14. How does this specification work in the context of a user agent’s Private \ Browsing or "incognito" mode?
+
+Like all other cookies in private browsers, partitioned cookies should be cleared when a private browsing session ends.
+Some browser private settings also block the use of cross-site cookies.
+User agents may wish to implement exceptions for partitioned cookies for these cross-site cookie restrictions for private browsing mode.
+
+### 2.15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+
+[Yes](https://github.com/WICG/CHIPS/blob/main/README.md#security-and-privacy-considerations).
+
+### 2.16. Does this specification allow downgrading default security characteristics?
+
+No.
+
+## 3. Threat Models
+
+### 3.1. Passive Network Attackers
+
+The Partitioned attribute can only be used for cookies being sent over secure connections, so the cookie strings are not visible to passive network attackers.
+
+### 3.2. Active Network Attackers
+
+See 3.1.
+
+### 3.3. Same-Origin Policy Violations
+
+By requiring the __Host- prefix, this proposal makes partitioned cookies scheme- and hostname-bound.
+However, even cookies with the __Host- prefix can still be sent to different ports (though this may change if [Origin-Bound Cookies](https://github.com/sbingler/Origin-Bound-Cookies) is enabled).
+
+### 3.4. Third-Party Tracking
+
+Partitioning cross-site cookies by top-level site is meant to reduce (or eliminate) third parties' ability to use cookies as a cross-site identifier.
+The goal of this proposal is for cross-site contexts to be able to associate a user's activity only within a single top-level site.
+
+### 3.5. Legitimate Misuse
+
+A third party may wish to use partitioned cookies to store a global user identifier that persists across multiple top-level domains.
+This identifier could be repeatedly derived on each top-level site a user visits using some side channel or fingerprinting techniques.
+Afterwards the third-party can store this identifier in a partitioned cookie so that when the user returns to each top-level site with the third-party embed, they no longer need to use the side channel to derive the cross-site identifier again.
+
+Though partitioned cookies could introduce a new place for third parties to store a derived cross-site identifier, a third-party script running in a top-level context could still store a cross-site identifier in the top-level site's cookie jar.
+Also, there are proposals for identifiers (e.g. [UID2.0](https://www.thetradedesk.com/us/knowledge-center/what-the-tech-is-unified-id-2-0)) which may disincentivize third parties from using partitioned cookies as a replacement to third-party cookies for cross-site tracking.
```

Comment:
I was thinking that UID2.0 is an example of an identifier that's "repeatedly derived on each top-level site a user visits", and which might be conveniently stored in a partitioned cookie, but I guess that standard also describes how to store it in the top-level site's cookie jar, so it's really an example of how the ecosystem has already gone through the whole flow you're describing.

No change needed.

Reply:
Ack'd